src/main/java/org/eclipse/yasson/internal/serializer/types/TypeSerializers.java

```diff
@@ -153,6 +154,17 @@ public static boolean isSupportedMapKey(Class<?> clazz) {
         return Enum.class.isAssignableFrom(clazz) || SUPPORTED_MAP_KEYS.contains(clazz);
     }
 
+    /**
+     * Whether type has a custom {@link JsonbSerializer} implementation.
+     *
+     * @param clazz        type to serialize
+     * @param jsonbContext jsonb context
+     * @return whether a custom JsonSerializer for the type is available
+     */
+    public static boolean hasCustomJsonbSerializer(Class<?> clazz, JsonbContext jsonbContext) {
+        return jsonbContext.getComponentMatcher().getSerializerBinding(clazz, null).isPresent();
```

Comment:
Should we be passing the `propertyCustomization` parameter from `SerializationModelCreator.createMapSerializer()` as the `null` parameter here?

Reply:
Hm, not sure here... This method is only to determine if there's a custom Serializer at all...